src/assets/config_files/usecases.json

```diff
@@ -111,6 +111,156 @@
         "next": "ta_form",
         "dataprotected": true,
         "files": []
+      },
+      {
+        "title": "Preparing a protein to run an atomistic Molecular Dynamics simulation",
+        "description": "Running a Molecular Dynamics setup from a protein in PDB format to obtain a system ready to be used as input for an MD simulation. Using GROMACS MD package and the building blocks library from the BioExcel project.",
+        "experience": ["all", "power"],
+        "maturity": ["beta"],
+        "disabled": false,
+        "picture": {
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/IRB_uc1.png"
+        },
+        "next": "ta_form",
+        "dataprotected": true,
+        "files": [
+          {
+            "entryname": "Protein atomistic MD simulation setup - Setup",
+            "appid": 175,
+            "contenttype": "x-ipynb+json",
+            "extension": ".ipynb",
+            "file": "73d225d1-3590-442b-8a5a-5210ba64af19",
+            "initial": true
+          },
+          {
+            "entryname": "Protein atomistic MD simulation setup",
+            "appid": 175,
+            "contenttype": "x-ipynb+json",
+            "extension": ".ipynb",
+            "file": "a49db178-9d3c-41e5-baab-ae819ab80bcf"
+          }
+        ]
+      },
+      {
+        "title": "Small molecule force field parameterization for atomistic Molecular Dynamics simulations",
+        "description": "Generate force field (GAFF) parameters for a small molecule in PDB format to use it in atomistic Molecular Dynamics simulations. Using ACPype, Ambertools and OpenBabel programs, and the building blocks library from the BioExcel project.",
+        "experience": ["all", "power"],
+        "maturity": ["beta"],
+        "disabled": false,
+        "picture": {
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/IRB_uc2.png"
+        },
+        "next": "ta_form",
+        "dataprotected": true,
+        "files": [
```

Comment:
No issues with this one. Everything works fine

Reply:
Great!